docs/sparkr.md

```diff
@@ -285,71 +285,57 @@ head(teenagers)
 
 # Machine Learning
 
-SparkR allows the fitting of generalized linear models over DataFrames using the [glm()](api/R/glm.html) function. Under the hood, SparkR uses MLlib to train a model of the specified family. Currently the gaussian and binomial families are supported. We support a subset of the available R formula operators for model fitting, including '~', '.', ':', '+', and '-'.
+SparkR supports the following Machine Learning algorithms.
 
-The [summary()](api/R/summary.html) function gives the summary of a model produced by [glm()](api/R/glm.html).
+* Generalized Linear Regression Model [glm()](api/R/glm.html)
+* Naive Bayes [naiveBayes()](api/R/naiveBayes.html)
+* KMeans [kmeans()](api/R/kmeans.html)
+* AFT Survival Regression [survreg()](api/R/survreg.html)
 
-* For gaussian GLM model, it returns a list with 'devianceResiduals' and 'coefficients' components. The 'devianceResiduals' gives the min/max deviance residuals of the estimation; the 'coefficients' gives the estimated coefficients and their estimated standard errors, t values and p-values. (It only available when model fitted by normal solver.)
-* For binomial GLM model, it returns a list with 'coefficients' component which gives the estimated coefficients.
+Under the hood, SparkR uses MLlib to train a model of the specified family. Currently the gaussian, binomial, Poisson and Gamma families are supported. We support a subset of the available R formula operators for model fitting, including '~', '.', ':', '+', and '-'.
```

Comment:
SparkR supports 4 MLlib machine learning models, the description here is only applicable to ```spark.glm```.